virt/vm_networking/virt-networking-overview.adoc

```diff
@@ -34,7 +34,7 @@ image:darkcircle-7.png[20,20] Secondary VM networks are typically bridged direct
 
 [NOTE]
 ====
-Connecting VMs directly to the underlay network is not supported on {product-rosa}.
+Connecting VMs to user-defined networks with the `layer2` topology is supported on public clouds.
```

Comment:
I would keep the old note. And add this new one below it. Just replacing "supported" with "recommended"